src/mesh/amr_loadbalance.cpp

```diff
@@ -488,7 +489,7 @@ void Mesh::RedistributeAndRefineMeshBlocks(ParameterInput *pin, int ntot) {
   MPI_Request *req_send, *req_recv;
   // Step 5. allocate and start receiving buffers
   if (nrecv != 0) {
-    recvbuf = new Real *[nrecv];
+    recvbuf = new ParArray1D<Real>[nrecv];
```

Comment:
Do you mean an array of `std::unique_ptr` to `ParArray1D` instead of an array of `ParArray1D` pointers?

Reply:
I just want to avoid having to call new and delete. 
Something like this...
```
std::unique_ptr<ParArray1D<Real>> sendbuf, recvbuf;

recvbuf = std::make_unique<ParArray1D<Real>> ( ParArray1D<Real>[nrecv] );

```

Reply:
why use pointers at all, smart or otherwise?

Reply:
Are you thinking of a 2D View?
If yes, this would "waste" memory as the buffers are of different size.

Reply:
oh it doesn't matter.  just ignore me and let's get this merged in! 😃

Reply:
Alright, let's revisit this later. I'm going to pull the trigger now.